custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -110,18 +110,17 @@ def handle_failure(self, response, payload_doc, repeat_record):
 def _get_person_case_properties(person_case_properties):
     state_choice = person_case_properties.get('current_address_state_choice', None)
     district_choice = person_case_properties.get('current_address_district_choice', None)
+    tu_choice = person_case_properties.get('tu_choice', None)
 
     if state_choice:
-        state_location = (SQLLocation.objects.
-                          get(location_id=person_case_properties['current_address_state_choice']))
+        state_location = SQLLocation.objects.get(location_id=state_choice)
     if district_choice:
-        district_location = (SQLLocation.objects.
-                             get(location_id=person_case_properties['current_address_district_choice']))
+        district_location = SQLLocation.objects.get(location_id=district_choice)
+
+    if tu_choice:
+        tu_location = SQLLocation.objects.get(tu_choice)
```

Comment:
hey, so I think we want to do what I mentioned on slack instead of this.
```py
person_owner = person_case_properties.get('owner_id')
if person_owner not in WEIRD_OWNER_IDS:
    phi_location = SQLLocation.objects.get(pk=person_owner)
    tu_location = phi_location.parent 
    district_location = phi_location.parent
    state_location = district_location.parent
```